appcli/templates/launcher.j2

```diff
@@ -63,7 +64,7 @@ function main()
         --volume "{{ path }}:{{ path }}" \
 {% endfor %}
         --volume "${DOCKER_SOCKET}:/var/run/docker.sock" \
-        --volume "${MOUNTED_DATA_DIR}/cli/home:/root" \
+        --volume "${MOUNTED_GENERATED_CONFIG_DIR}/cli/home:/root" \
```

Comment:
This isn't great.
A home directory could contain actual data depending on what the system does.
What if something in the app writes data into the home dir that we need in future? We will lose it if the config dir gets regenerated.

Can we instead copy the required helm stuff from the generated directory into the home directory once container is launched?